codeship-steps.yml

```diff
@@ -0,0 +1,7 @@
+- name: push_to_ecr
+  service: valhalla
+  type: push
+  image_name: 265147712162.dkr.ecr.us-east-1.amazonaws.com/gaiagps/valhalla
```

Comment:
codeship-services.yml has a different image name than this does, they should probably match.